Doc/library/sndhdr.rst

```diff
@@ -53,3 +53,51 @@ be the sample size in bits or ``'A'`` for A-LAW or ``'U'`` for u-LAW.
    .. versionchanged:: 3.5
       Result changed from a tuple to a namedtuple.
 
+The following sound header types are recognized, as listed below with the return value
+from :func:`whathdr`: and :func:`what`:
+
++------------+------------------------------------+
+| Value      | Sound header format                |
++============+====================================+
+| ``'aifc'`` | Compressed Audio Interchange Files |
++------------+------------------------------------+
+| ``'aiff'`` | Audio Interchange Files            |
++------------+------------------------------------+
+| ``'au'``   | AU Files                           |
++------------+------------------------------------+
+| ``'hcom'`` | HCOM Files                         |
++------------+------------------------------------+
+| ``'sndr'`` | SNDR Files                         |
++------------+------------------------------------+
+| ``'sndt'`` | SNDT Files                         |
++------------+------------------------------------+
+| ``'voc'``  | VOC Files                          |
++------------+------------------------------------+
+| ``'wav'``  | WAV Files                          |
++------------+------------------------------------+
+| ``'8svx'`` | 8SVX Files                         |
++------------+------------------------------------+
+| ``'sb'``   | SB Files                           |
++------------+------------------------------------+
+| ``'ub'``   | UB Files                           |
++------------+------------------------------------+
+| ``'ul'``   | uLAW Audio Files                   |
++------------+------------------------------------+
+
+.. data:: tests
+
+   A list of functions performing the individual tests.  Each function takes two
+   arguments: the byte-stream and an open file-like object. When :func:`what` is
+   called with a byte-stream, the file-like object will be ``None``.
+
+   The test function should return a string describing the image type if the test
+   succeeded, or ``None`` if it failed.
+
+Example::
+
+   >>> import sndhdr
+   >>> imghdr.what('bass.wav')
+   'wav'
+   >>> imghdr.whathdr('bass.wav')
+   'wav'
```

Comment:
When using "Python console" code blocks, we can use a special `pycon` to better highlight the syntax:


```suggestion
Example:

.. code-block:: pycon

   >>> import sndhdr
   >>> imghdr.what('bass.wav')
   'wav'
   >>> imghdr.whathdr('bass.wav')
   'wav'
```